docs/mail.rst

```diff
@@ -0,0 +1,88 @@
+Email basics
+============
+
+The big picture
+---------------
+Sending email involves lots of components.
+
+.. image:: https://res.cloudinary.com/shaka/image/upload/v1542098296/ansible_mailserver/simple_mail_1.png
```

Comment:
I have two remarks on this schema : 
* It's not clear that the first answer from the database is a dashed line
* Haven't you changed "obiwan@jedi" in all of your tests ?

Reply:
I was not clear enough, you reviewed wikipedia :laughing:, it's all my fault.

Reply:
I corrected both :+1: